src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -44,7 +44,6 @@ interface SearchBarInjectedDeps {
 
 export interface SearchBarOwnProps {
   indexPatterns?: IIndexPattern[];
-  isLoading?: boolean;
```

Comment:
Should this have been removed? Is it not being used anywhere the search bar is being used?